src/parser.c

```diff
@@ -1,9 +1,8 @@
-/* A Bison parser, made by GNU Bison 3.3.2.  */
+/* A Bison parser, made by GNU Bison 3.0.4.  */
```

Comment:
Did you mean to check in the generated code?  Do some builds start with parser.c, rather than parser.y?

Reply:
Yes, I did.  We commit Bison outputs for users who build from source on systems that lack a recent Bison.  I should not change the Bison version though...